WcaOnRails/app/inputs/date_picker_input.rb

```diff
@@ -3,10 +3,10 @@ def input(wrapper_options)
     set_html_options
     set_value_html_option
 
-    template.content_tag :div, class: 'input-group date datetimepicker' do
+    #template.content_tag :div, class: 'input-group date datetimepicker' do
       input = super(wrapper_options) # leave StringInput do the real rendering
-      input_button + input + utc_addon
-    end
+    #  input_button + input + utc_addon
```

Comment:
Commented out code =(. How are users going to know that times are in UTC? Can we leave the UTC addon, or make it obvious some other way?